ruler-common/src/main/java/com/spotify/ruler/common/veritication/VerificationException.kt

```diff
@@ -0,0 +1,20 @@
+/*
+* Copyright 2024 Spotify AB
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package com.spotify.ruler.common.veritication
+
+class VerificationException(label: String, size: Long, threshold: Long) :
```

Comment:
nit: Wouldn't SizeExceededException make it more clear?

Reply:
Agreed! I've renamed the class. 🙌🏽